tests/suites/0_stateless/09_fuse_engine/09_0014_func_clustering_information_function.sql

```diff
@@ -0,0 +1,14 @@
+-- Create table t09_0014
+create table t09_0014(a int, b int) cluster by(b,a);
+
+insert into t09_0014 values(0,3),(1,1);
+insert into t09_0014 values(1,3),(2,1);
+insert into t09_0014 values(4,4);
+
+select *  from t09_0014 order by b, a;
+
+--Bug in cluster mode: https://github.com/datafuselabs/databend/issues/5473
```

Comment:
We can skip the test in cluster mode with a '_cluster' suffix new test file, let's run only in the stand-alone mode.